production/loki-mixin-compiled-ssd/dashboards/loki-operational.json

```diff
@@ -4825,7 +4825,7 @@
                   "steppedLine": false,
                   "targets": [
                      {
-                        "expr": "sum(rate(cortex_dynamo_failures_total{cluster=\"$cluster\", namespace=\"$namespace\"}[5m]))",
+                        "expr": "sum(rate(loki_dynamo_failures_total{cluster=\"$cluster\", namespace=\"$namespace\"}[5m]))",
```

Comment:
The dynamo and cassandra metrics have had a loki prefix for a while now. I assume the dashboards were never changed. They aren't used much so that's probably why nobody noticed,